client/whetstone.cpp

```diff
@@ -98,7 +98,7 @@ int whetstone(double& flops, double& cpu_time, double min_cpu_time) {
     SPDP x,y,z;
     long j,k,l, jjj;
     SPDP e1[4];
-    double startsec, finisec;
```

Comment:
This is not related to this PR.
Please create a separate branch and a separate PR with this change.